migrations/2023-10-19-102636_back_fill_n_remove_connector_response/down.sql

```diff
@@ -0,0 +1,2 @@
+
+ALTER TABLE payment_attempt DROP COLUMN authentication_data, DROP COLUMN encoded_data;
```

Comment:
will two drop commands work in a single alter statement?

Reply:
As you can see migration check as passed.